kernel/uint63_31.ml

```diff
@@ -58,6 +58,15 @@ let lt x y =
 let le x y =
   Int64.compare x y <= 0
 
+    (* signed comparison *)
+ (* we shift the arguments by 1 to the left to that the top-most bit is interpreted as a sign *)
+ (* the zero at the end doesn't change the order (it is sable by multiplication by 2) *)
```

Comment:
```suggestion
 (* we shift the arguments by 1 to the left so that the top-most bit is interpreted as a sign *)
 (* the zero at the end doesn't change the order (it is stable by multiplication by 2) *)
```